src/pages/heedongChoi/Login/LoginHee.js

```diff
@@ -1,7 +1,87 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import '../../heedongChoi/Login/LoginHee.scss';
 
-const Login = () => {
-  return <div></div>;
-};
+function LoginHee() {
+  const [userInfo, setUserInfo] = useState({
+    email: '',
+    password: '',
+  });
 
-export default Login;
+  const navigate = useNavigate();
+  const goToMain = () => {
+    validCheck ? navigate('/hee/Main') : alert('똑바로 써주세요');
+  };
+
+  // const goToMainToken = () => {
+  //   fetch('http://10.58.1.230:8000/users/signin', {
+  //     method: 'POST',
+  //     body: JSON.stringify({
+  //       email: userInfo.email,
+  //       password: userInfo.password,
+  //     }),
+  //   })
+  //     .then(res => res.json())
+  //     .then(res => {
+  //       if (res.token) {
+  //         localStorage.setItem('login-token', res.token);
+  //         navigate('/hee/Main');
+  //       } else {
+  //         alert('땡ㅋㅋ');
+  //       }
+  //     });
+  // };
+  const validCheck =
+    userInfo.email.includes('@') && userInfo.password.length > 5;
+
+  function handleUserInput(event) {
+    const { name, value } = event.target;
+    setUserInfo(userObject => ({
+      ...userObject,
+      [name]: value,
+      [name]: value,
```

Comment:
```suggestion
      [name]: value,
```
해당 부분은 한번만 사용해도 될 것 같습니다

Reply:
라이브 리뷰때 잘 설명해주셔서 감사합니다